bcipy/helpers/tests/test_triggers.py

```diff
@@ -13,67 +16,56 @@
     extract_from_copy_phrase,
     read_triggers,
     LslCopyPhraseLabeller,
-    NONE_VALUES,
     trigger_durations,
     write_trigger_file_from_lsl_calibration,
 )
-from bcipy.signal.generator.generator import gen_random_data
+from bcipy.helpers.raw_data import sample_data, write
 
 import psychopy
 
 
-def sample_raw_data(trigger_seq: List[Tuple[str, str]] = [],
-                    first_trg_time: int = 100,
-                    trigger_interval: int = 10,
-                    daq_type: str = 'TestStream',
-                    sample_rate: int = 300,
-                    ch_names: List[str] = ['c1', 'c2',
-                                           'c3']) -> Tuple[str, List[float]]:
-    """Helper function for creating mock data that looks like the raw_data.csv
-    output. Adds trigger data to the TRG column at the specified interval.
+def write_sample_raw_data(raw_data_path: str,
```

Comment:
write_sample_trigger_data? We can reserve RawData as a concept for acquisition data output.